docs/elements/sapc4c-hd/index.md

```diff
@@ -0,0 +1,67 @@
+---
+heading: SAP C4C Helpdesk
+seo: Overview | SAP C4C | Cloud Elements API Docs
+title: Overview
+description: Integrate SAP C4C into your application via the Cloud Elements APIs.
+layout: sidebarelementdoc
+breadcrumbs: /docs/elements.html
+elementId: 3522
+parent: Back to Element Guides
+order: 1
+sitemap: false
+---
+
+# Welcome to the {{page.heading}} Element
+
+{% include callout.html content="<strong>On this page</strong></br><a href=#element-details>Element Details</a></br><a href=#base-url>Base URL</a></br><a href=#authenticating-with-cloud-elements>Authenticating with Cloud Elements</a></br><a href=#error-codes>Error Codes</a>" type="info" %}
+
+# Element Details
+
+| Element Information | Details     |
+| :------------- | :------------- |
+| API Documentation | [{{page.heading}} API  documentation](http://help-legacy.sap.com/saphelp_sapcloudforcustomer/en/PUBLISHING/IntegrationServices.html) |
+| Authentication | Basic  |
+| Events | Polling |
+| Bulk | Supported |
```

Comment:
I think it's a great idea. However, I looked through their docs and couldn't find anything about it. If you take a look at the bulk file, you'll see I made a vague reference to checking with SAP about rate limits.